tests/test-cmd/test.run.js

```diff
@@ -281,6 +281,7 @@ describe('run', () => {
       const args = {
         addonId: 'some-addon@test-suite',
         client,
+        // $FLOW_IGNORE: fake a Firefox ChildProcess using an EventEmitter for testing reasons.
```

Comment:
I think we could solve this problem by switching:

````
import type {ChildProcess} from 'child_process';
````

to an [interface](https://flowtype.org/docs/quick-reference.html#interfaces):

````
interface ChildProcess {
  on: () => ...,
  stderr: {
    on: () => ...,
  },
  stdout: {
    on: () => ....,
  }
}
````

It's a little nicer to simply import the type directly from `child_process` but using interfaces will give us a lot of powerful flexibility in testing.